updater

```diff
@@ -561,7 +558,7 @@ update_now_gui() { #input: updatable_files and updatable_apps variables
   
   #contruct a queue to send to terminal_manage_multi.
   if [ ! -z "$updatable_files" ]; then
-    queue+="update filelist:$(echo -n "$updatable_files" | tr '\n' ':')"$'\n'
+    queue+="$(echo "$updatable_files" | sed 's/^/update-file /g')"$'\n'
```

Comment:
same thing as before. updating one file with changes that are not compatible with the repo is not good practice even if your next commit completes the changes.  just make one commit next time

Reply:
Is there a way to commit to two files simultaneously on the github.com web interface?

Reply:
I would normally use `git col` but I'm basing off my local install which is based on `master` and I'd rather not switch branches.

Reply:
> Is there a way to commit to two files simultaneously on the github.com web interface?

yup, for about 2 years now. dropdown next to edit, and open in github.dev. its basically vscode in the browser